src/news_app/clear_feed.js

```diff
@@ -0,0 +1,5 @@
+export const clearFeed = () => {
```

Comment:
I think *_feed files can be in just one file did you follow any naming convention for this file ? like this one ? https://google.github.io/styleguide/jsguide.html#file-name